WizardModeSelection: navigate with keyboard; enable accessibility

rating89us opened the pull request.

selsta commented:
Sorry, again, I don't know how to review this :D If I'm on the wizard mode selection page, how can I select anything with the keyboard?

rating89us commented:
Press _Tab_ to select the first item. Then, _Tab_ will select the next item and _Shift + Tab_ will select the previous item.

rating89us commented:
should be working now

selsta commented:
I can't get this one specifically to work.

rating89us commented:
Now it's working. Please test again.

selsta commented:
Looking good now.

selsta reviewed (changes requested):
Clicking on "Simple mode" or "Simple mode bootstrap" and then on back again breaks selecting with tab.

rating89us commented:
> Clicking on "Simple mode" or "Simple mode bootstrap" and then on back again breaks selecting with tab.

Fixed and added "About remote node" subpages

selsta commented on wizard/WizardMenuItem.qml:
```diff
@@ -81,9 +98,7 @@ RowLayout {
-            onClicked: {
-                rowlayout.menuClicked();
-            }
+            onClicked: rowlayout.menuClicked() + (!checkbox ? rowlayout.focus = false : "")
```
It might be better to split this up.
